QND.Hyperv2012R2/details.json

```diff
@@ -5,6 +5,7 @@
   "Version": "1.1.0.72",
   "Author": "Daniele Grandini",
   "IsFree": true,
+  "Description": "An Hyperv 2012 R2 Operations Manager Management Pack community backed",

```

Comment:
Could do with reworking.. perhaps "A community-backed Hyper-V 2012 R2 Operations Manager MP"

Reply:
Sounds much better, much of the beta data was part of a copy & paste storm, I'll get this update in place.